### ISSUE TITLE
[ENH] Update pip and conda packages

### ISSUE THREAD

**Issue by kikocorreoso** [OpenHydrology/lmoments3#8]
Related (#5 #3 #6).

Pypi and anaconda packages need to be updated in order to use `lmoments3` with modern versions of `scipy` (see #6) and Python (see #3 and #5).

Only the people with permissions on pypi and anaconda could update the packages.

If the package is not maintained I could fork and maintain it but it would be better to keep the work in the original repo and channels (github, pypi, anaconda,...).

**Comment by neilnutt** [comment]
If you were to maintain, what is your preference? I can give you admin rights here but I don't think I can at pypi or Anaconda.

**Comment by kikocorreoso** [comment]
Hi, to update PyPI and anaconda we should have rights to upload packages on those channels:

* https://pypi.org/project/lmoments3/
* https://anaconda.org/OpenHydrology

or ping @faph to help on this.

If @faph is not available I think we could try these options:

*  we could upload lmoments3 to the conda-forge channel but on Pypi we should change the name of the package to avoid conflicts (this would be a mess -> old `lmoments3` on openhydrology anaconda channel, new `lmoments3` on conda-forge anaconda channel, new `lmoments3_name` on PyPI, `lmoments3` repo on github,...).
* We could fork lmoments3 and continue development (mainly maintenance) on other repository using a new name (`lmom3` , `py-lmoments3`, ...).
* Other options...

Let's see if @faph can help on this or try to decide one of the potential alternatives.

@neilnutt Thanks for taking care on this.

**Comment by kikocorreoso** [comment]
It seems @faph is not available.

cc @neilnutt @monocongo 

I have a `lmoments3` dependency on `scikit-extremes`. It seems @monocongo also has this dependency.

I could just ship the necessary files from `lmoments3` to make `scikit-extremes` work as it is a very tiny lib (with attribution and license).

But I think it would be better a fork.

Should we fork this repo, rename the library and continue the development there so we can update PyPI, conda,...? I think none of us have access or is a maintainer of `lmoments3` on PyPI nor conda.

**Comment by monocongo** [comment]
I gave up on `lmoments3` due to weird errors that I no longer get from my own implementations (also based on Hosking etc.). Your idea to fork and rename in order to move forward in the right way sounds like a winner.

**Comment by kikocorreoso** [comment]
@monocongo 

Is your code in a standalone lib or is it integrated as part of other library?

Could your code be integrated in a potential fork of `lmoments3` to make a better standalone lmoments estimation library?

**Comment by kikocorreoso** [comment]
Oh, maybe is this one? https://github.com/monocongo/climate_indices/blob/master/climate_indices/lmoments.py

**Comment by monocongo** [comment]
Yes, that's my code that uses L-moments. Bear in mind that I don't even pretend to be a statistician and have only a tentative grasp of what's being computed there, but it seems to match very closely with the `lmoments3` results without resulting in dubious "invalid L-moment" errors for my climate indices calculations. If someone more knowledgable can go through that code and confirm its veracity I will be grateful.

**Comment by huard** [comment]
See https://github.com/scipy/scipy/issues/11782#issuecomment-636111912
